### PR TITLE
[ADD] dead_mans_switch_therp

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by hbrunn** [comment]
@lfreeke @NL66278 @gfcapalbo ping

**Comment by NL66278** [comment]
Merge contains a pyc file. No .gitignore yet?

File: dead_mans_switch_therp/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Dead man's switch for Therp",
+    "version": "1.0.0",
```
Comment (NL66278) [inline dead_mans_switch_therp/__openerp__.py]: Put 6.1 in front. And maybe use shortened headers?

**Comment by hbrunn** [comment]
@NL66278 thanks, `.gitignore` added, pyc removed and done as you said

**Comment by hbrunn** [comment]
ping

**Comment by hbrunn** [comment]
ping

**Comment by hbrunn** [comment]
@Therp/core please review/merge

**Pull request merged**